config/default.json

```diff
@@ -74,7 +83,10 @@
     "helmetEnabled": false,
     "termsAndConditionsEnabled": false,
     "redisEnabled": false,
-    "oidcEnabled": false
+    "oidcEnabled": false,
+    "workAllocationEnabled": false,
+    "roleEnabled": false,
+    "caseworkerRefEnabled": false
```

Comment:
Can remove the "roleEnabled" and "caseworkerRefEnabled" as not necessary (I could do that later)